clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -127,4 +127,27 @@ private EhcacheEntityResponse invoke(EhcacheEntityMessage message) throws Except
       }
     }
   }
+
+  public EhcacheEntityResponse invokeCompleted(EhcacheEntityMessage message) throws Exception {
```

Comment:
I get the idea of this `invokeCompleted` method but I have two problems with it:
1) the name is quite poor
2) it duplicates 99% of the `invoke` method's code